docs/api/vi.md

```diff
@@ -224,6 +224,10 @@ vi.mock('./example.js')
 test('1 + 1 equals 10', async () => {
   vi.mocked(example.calc).mockReturnValue(10)
   expect(example.calc(1, '+', 1)).toBe(10)
+
+  // To partially mock `function someFn(): { prop1: string, prop2: number }`
```

Comment:
I don't think this makes it clear, to be honest; I am still confused with the wording. Maybe we shouldn't imply anything and just say that it allows mocking the function return to be a partial object instead of the whole object. We should also make it clear that this applies only to types

Reply:
I feel this `partial` feature is a niche and it's hard to explain in general. It would've been nice if OP have included a convincing example :sweat_smile: https://github.com/vitest-dev/vitest/issues/1739

I'll try to adjust the example, but I'm not sure if we can do better than that. If `vi.mocked` is already explained as type helper, so I just wish users would understand.